### ISSUE TITLE
Tumor signet ring cells

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10066]
**Preferred term label:**
Tumor signet ring cells

**Synonyms**

**Definition (free text, please give PubMed ID)**
An example is https://www.ncbi.nlm.nih.gov/medgen/?term=C1335170

**Parent term (use hpo.jax.org/app)**
HPO has https://hpo.jax.org/app/browse/term/HP:0032175, which has a very similar name but an unrelated concept.

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://www.ncbi.nlm.nih.gov/books/NBK482284/

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
the HPO wants to extend out to pathological findings and has one new branch [Biospecimen phenotypic feature](https://www.ebi.ac.uk/ols4/ontologies/hp/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FHP_0020228?lang=en)
This branch is appropriate if one is phenotyping a tumor or biospecimen.
If one is recording the observation of a Krukenberg tumor in a patient, then the following term would be appropriate

Krukenberg tumor
def: Krukenberg tumor is a metastatic signet-ring cell adenocarcinoma of the ovary characterized by mucin-rich, signet-ring cells.
comment: Krukenberg tumors originate from a gastrointestinal primary in 70% of cases, but also involving the colon, breast, appendix, and biliary tract. Krukenberg tumor is characterized by mucin-laden signet ring cells with offset, hyperchromatic nuclei. The cytoplasmic mucin within signet ring cells is necessary for diagnosis. The cytoplasm of the signet ring cells may be diffusely granular with eosinophilic bodies, largely vacuolated, or a combination featuring a large vacuole containing a drop of mucin, creating an eosinophilic body. Many patterns of signet ring cell organization are possible even within the same tumor. 
PMID: [29489206](https://pubmed.ncbi.nlm.nih.gov/29489206)

It is probably better to go with the above term for now, since the pathology findings will be a big lift and we want to do it systematically with workshops.
Thoughts?

**Comment by MickeySegal** [comment]
But also:
https://www.ncbi.nlm.nih.gov/books/NBK541116/
High-grade mucinous carcinoma peritonei with signet ring cells: Any lesion with a component of signet ring cells, classified separately because of their worse prognosis

**Comment by pnrobinson** [comment]
@MickeySegal 
Sorry for the delay. In the meantime, we have recently started to model biopsies separately, and this this term now "fits". I have added
HP:0020168
Tumor signet ring cell
A signet ring has a flattened surface at the top of the ring with an engraved or raised symbol, historically used to stamp a seal into correspondence. Comparatively, these cells acquire a signet cell morphology due to accumulation of abundant mucin in cytoplasm, leading to nuclei dislocation to the periphery.
